manual/manual/html_processing/scss/manual.scss

```diff
@@ -0,0 +1,311 @@
+}
+ul{
+    list-style-position:outside
+}
+ul>li{
+    margin-left:22px
+}
+li>:first-child{
+    margin-top:0
+}
+.left{
+    text-align:left
+}
+.right{
+    text-align:right
+}
+a{
+    text-decoration:none;
+    color:#92370a
+}
+a:hover{
+    box-shadow:0 1px 0 0 #92370a
+}
+:target{
+    background-color:rgba(255,215,181,.3)!important;
+    box-shadow:0 0 0 1px rgba(255,215,181,.8)!important;
+    border-radius:1px
+}
+:hover>a.section-anchor{
+    visibility:visible
+}
+a.section-anchor:before{
+    content:"#"
+}
+a.section-anchor:hover{
+    box-shadow:none;
+    text-decoration:none;
+    color:#555
+}
+a.section-anchor{
+    visibility:hidden;
+    position:absolute;
+    margin-left:-1.3em;
+    font-weight:400;
+    font-style:normal;
+    padding-right:.4em;
+    padding-left:.4em;
+    color:#d5d5d5
+}
+.h10,.h7,.h8,.h9,h1,h2,h3,h4,h5,h6{
+    font-family:"Fira Sans",Helvetica,Arial,sans-serif;
+    font-weight:400;
+    margin:.5em 0 .5em 0;
+    padding-top:.1em;
+    line-height:1.2;
+    overflow-wrap:break-word
+}
+h1{
+    font-weight:500;
+    font-size:2.441em;
+    margin-top:1.214em
+}
+h1{
+    font-weight:500;
+    font-size:1.953em;
+    box-shadow:0 1px 0 0 #ddd
+}
+h2{
+    font-size:1.563em
+}
+h3{
+    font-size:1.25em
+}
+h1 code{
+    font-size:inherit;
+    font-weight:inherit
+}
+h2 code{
+    font-size:inherit;
+    font-weight:inherit
+}
+h3 code{
+    font-size:inherit;
+    font-weight:inherit
+}
+h3 code{
+    font-size:inherit;
+    font-weight:inherit
+}
+h4{
+    font-size:1.12em
+}
+.ocaml,.pre,code,pre,tt{
+    font-family:"Fira Mono",courier;
+    font-weight:400
+}
+.pre,pre{
+    border-left:4px solid #e69c7f;
+    overflow-x:auto;
+    padding-left:1ex
+}
+li code,p code{
+    background-color:#f6f8fa;
+    color:#0d2b3e;
+    border-radius:3px;
+    padding:0 .3ex
+}
+.pre .code,.pre.code,pre code{
+    background-color:inherit
+}
+p a>code{
+    color:#92370a}
+.pre code.ocaml,.pre.code.ocaml,pre code.ocaml{
+    font-size:.893rem}
+.keyword,.ocamlkeyword{
+    font-weight:500}
+section+section{
+    margin-top:25px}
+ul.itemize{
+    list-style:none}
+ul.itemize li::before{
+    content:"▶";
+    color:$logocolor;
+    font-size:smaller;
+    margin-right:4px;
+    margin-left:-1em}
+
+/* Table of Contents in the Left-hand sidebar */
+nav.toc{
+    @include nav-toc;
+    &.brand{
+	@include brand;
+    }
+    .toc_title{
+	display:block;
+	margin:.5em 0 1.414em}
+/* .toc_title a{ */
+/*     color:#777; */
+/*     font-size:1em; */
+/*     line-height:1.2; */
+    /*     font-weight:500} */
+
+}
+.tableau {
+    table {
+	border-collapse: collapse;
+    }
+    td {
+	background:#f8f7f6;
+	border:1px solid #ccc;
+	padding-left:3px;
+	padding-right:3px;
+    }
+}
+
+/* Mobile */
+@media only screen and (max-width:95ex){
+    .content{
+	margin:auto;
+	padding:2em
+    }
+    nav.toc{
+	@include nav-toc-mobile;}
+}
+
+.pre,pre{
+    background:linear-gradient(to left,#fff 0,#ede8e5 100%)
+}
```

Comment:
Note that before merging this PR, we should discuss this gradient. I really dislike it when reading code block. In particular on wide examples, this gradient alters the contrast and the visibility of the code depending on the position in the code block. Should we open a poll?

Reply:
a poll could be useful if we restrict participants to beginners, since after all the "tutorial" part is for them.
But maybe it's not worth the trouble. My mission for the OCaml foundation is to make the website look more modern and attractive (and useful of course), and gradients are "back in fashion" since a couple of year; but if you think that's not good for code block, no problem I will remove them

Reply:
My issue is really only with this gradient on a large text-area. The other gradient looks fine.

Reply:
It's not easy to tell which ones are large just using the CSS. I did remove gradients for`.pre`. Is this OK with you or did you mean anything else?
(in which case it might be easier for the discussion to explicitly mention which page)